src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/user/RespondToClaimCallbackHandler.java

```diff
@@ -77,6 +80,19 @@ private CallbackResponse validateUnavailableDates(CallbackParams callbackParams)
             .build();
     }
 
+    private CallbackResponse validateExperts(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        var experts = caseData.getRespondent1DQ().getExperts();
+        List<String> errors = new ArrayList<>();
+        if (experts.getExpertRequired() == YES && experts.getDetails() == null) {
```

Comment:
thoughts on making this a method in Experts class?

Reply:
it's simple enough I'd keep it in validator